dsrc_driver/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>dsrc_driver</name>
-  <version>1.0.0</version>
+  <version>2.8.2</version>
```

Comment:
Is it necessary to version the driver the same as CARMAPlatform instead of 1.1? Will we need to update the version in every driver when we update CARMA?

Reply:
No, we can version it however. If it makes sense to return this to version 1.0.0 I can do so, I think it does.